packages/cli/test/scripts/balance.test.js

```diff
@@ -2,16 +2,19 @@
 require('../setup');
 
 import utils from 'web3-utils';
-import { ZWeb3, Contracts } from '@openzeppelin/upgrades';
+import { Contracts } from '@openzeppelin/upgrades';
+import { accounts, web3 } from '@openzeppelin/test-environment';
+
 import CaptureLogs from '../helpers/captureLogs';
+import { fromWei } from '../../src/utils/units';
 
 import balance from '../../src/scripts/balance';
 
 const ERC20 = Contracts.getFromLocal('ERC20Fake');
 const ERC20Detailed = Contracts.getFromLocal('ERC20FakeDetailed');
 
-contract('balance script', function(accounts) {
-  const [_, accountAddress] = accounts.map(utils.toChecksumAddress);
+describe('balance script', function() {
+  const [accountAddress] = accounts.map(utils.toChecksumAddress);
```

Comment:
```suggestion
  const [accountAddress] = accounts;
```

`test-environment` exports checksummed accounts, so we can drop this

Reply:
Note that multiple test files follow this same pattern.

Reply:
Nice. I was thinking we should do checksum at `test-env` but it turns out we already do it.